### ISSUE TITLE
No way to follow Jira inline comment

### ISSUE THREAD

**Issue by pholat** [cmcaine/tridactyl#212]
When there is yellow marked inline comment it can't be followed by `f`. As it's a span, it's highly possible that there is JS reacting on onpress action.

Is there a way here to browse html text and emulate mouse press / enter on selected text? I believe that I've read during last week that there is no feature to browse page text in tridactyl ( kind like it worked in vimperator visual mode ) I would be grateful for clarification.

**Comment by cmcaine** [comment]
There is no visual mode as yet. You might have some luck with Firefox's caret mode (F7).

Can you be specific about what exactly happened and what you expected to happen?

I think you pressed `f`, the inline comment was hinted, you selected it and then nothing happened, and that you expected to be taken to a new page.

An example URL featuring the issue would be very helpful.

`f` does simulate a mouse click, but perhaps not well enough. You can check it out in `src/hinting.ts:simulateClick` and `src/dom.ts:mouseEvent`

**Comment by pholat** [comment]
Thanks, I was trying to find some open confluence page quickly but failed. 

I pressed `f` but it wasn't hinted, though it was clickable. Then I checked and saw that it's a span of some css class, which is why I thought it works on some JS onclick action. Then I thought about using visual vimperator mode, but now I know it's not implemented yet. 

Great thanks for clarification! I'll gladly read into implementation. Unfortunately my JS/TS skills are not there yet to implement the mode.
